webportal/src/app/cluster-view/hardware/hardware.component.ejs

```diff
@@ -1 +1,65 @@
-<iframe style="display:block; width:100%; height:90vh; border:none;" src="<%= grafanaUri %>/dashboard/db/pai_nodeview"></iframe>
\ No newline at end of file
+<section class="content-header"><%= breadcrumb({ breadcrumbTitle: "Hardware Utilization" }) %></section>
+
+<section class="content">
+<div class="box">
+    <div class="box-body">
+        <table id="hardware-table" class="table table-bordered table-striped" style='vertical-align:middle'>
+        <thead>
+            <tr>
+                <th>Machine Name</th>
+                <th>IP Address</th>
+                <th style='width:80px; text-align:center'>CPU</th>
```

Comment:
pls write a scss file instead of inline styles

Reply:
#Solved.